CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add configurable `apiServer.certSANs`.
 - Add machine health checks for worker nodes.
 
+### Changed
+
+- Values schema: applied normalization using `schemalint normalize`
```

Comment:
There shouldn't be any changelog here I think. Is it relevant for the user?

Reply:
No, it's not relevant to users. I'll remove it.